docs/sphinx/source/whatsnew.rst

```diff
@@ -6,6 +6,14 @@ What's New
 
 These are new features and improvements of note in each release.
 
+Bug fixes
+~~~~~~~~~
+* Fixed a plotting issue in the IV curve gallery example (:pull:`1895`)
+
+Contributors
+~~~~~~~~~~~~
+* Will Hobbs (:ghuser:`williamhobbs`)
+
```

Comment:
Can you move these changes to [v0.10.3.rst](https://github.com/williamhobbs/pvlib-python/blob/iv_gallery_plot_fix/docs/sphinx/source/whatsnew/v0.10.3.rst)?  `whatsnew.rst` is just an index for the files for each release; it doesn't house any actual content itself.

Also, I think this item should be under `Documentation` instead of `Bug fixes`.

Reply:
Looks like you're using GitHub's web interface to make commits?  I don't use it myself but I assume it's possible to edit that file and create a new commit that removes those lines, in much the same way it created a commit to add them.  

Another option is to [suggest a change](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) to delete those lines and then [accept it](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request) here in the PR itself.  

The most general method is to use a local git client and your own text editor, but that's a bit more involved than clicking a few buttons here on github :)